### DISCUSSION TITLE
How to place a cursor on each selected line?

### DISCUSSION THREAD

**Discussion by xthezealot** [zed-industries/zed#12361] [Feature Requests]
When multiple lines are selected, how to place a new cursor on each line?

In VS Code, there is the `Alt + Shift + I` shortcut.
I cannot find the equivalent in [the doc](https://zed.dev/docs/key-bindings?highlight=shortcuts#all-key-bindings).

**Comment by xthezealot** [comment]
OK, I just found there is the "split selection into line" command.
But it seems to be no key binding for it.
Here is mine:
```
[
  {
    "context": "Editor",
    "bindings": {
      "cmd-alt-l": "editor::SplitSelectionIntoLines"
    }
  }
]
```

**Reply by CharlesLemon** [reply]
Thanks!
